.claude/scripts/review-provider-loop-contract.test.sh

```
@@ -156,10 +158,45 @@ assert_prose "${constitution}" 'an empty object is a reply container, never a re
+# green and sent the run down into weekly-limited Codex and monthly-limited Bugbot.
+#
+# Both halves are pinned deliberately. The ACCEPT half alone would still pass if a later edit
+# re-broadened the rule to every command reply, and the REJECT half alone would still pass if the
+# third artifact were dropped again — so each surface asserts that it accepts a verdict-bearing
+# reply AND that a bare acknowledgement remains a non-review.
+for f in "${constitution}" "${surveyor}"; do
+  assert_prose "${f}" 'command-invocation reply comment carrying a verdict' \
+    "$(basename "${f}") does not accept CodeRabbit's verdict-bearing command reply, so a real green at head reads as none"
+  assert_prose "${f}" 'I found no actionable issues' \
+    "$(basename "${f}") does not pin the verdict line that discriminates a review from an acknowledgement"
+  assert_prose "${f}" 'prefix of `headRefOid`' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)